_docs/_latest/configure/gsuite.md

```diff
@@ -5,27 +5,27 @@ order: 600
 
 #  {{ page.title }}
 
-This page describes how to enable the data collection of G Suite Google Groups for
-processing by Forseti Inventory.
+This page describes how to enable the data collection of G Suite for processing by Forseti Inventory.
 
 ---
 
 ## Enable Domain-Wide Delegation in G Suite
 
-To enable collection of G Suite Google Groups, follow the steps below to enable your existing forseti
+To enable collection of G Suite data, follow the steps below to enable your existing forseti
 service account for this functionality. Read more about 
 [domain-wide delegation](https://developers.google.com/identity/protocols/OAuth2ServiceAccount?hl=en_US#delegatingauthority).
 
 ### Enable DwD on a service account
 
 1. Navigate to the [service account](https://pantheon.corp.google.com/projectselector/iam-admin/serviceaccounts){:target="_blank"} page
 
-   * Click on the 3 dots on the right hand side of your Forseti GCP Server service account and select **Edit**
-   {% responsive_image path: images/docs/configuration/service_account_edit.png alt: "Service Account Edit" indent: 3 %}
+   1. Click on the 3 dots on the right hand side of your Forseti GCP Server service account and select **Edit**
    
-   * Select the **Enable G Suite Domain-wide Delegation** checkbox and click **SAVE**
+      {% responsive_image path: images/docs/configuration/service_account_edit.png alt: "Service Account Edit" indent: 3 %}
+   
+   1. Select the **Enable G Suite Domain-wide Delegation** checkbox and click **SAVE**
             
-{% responsive_image path: images/docs/configuration/service_account_enable_dwd.png alt: "Service Account Enable DwD" indent: 3 %}
+   {% responsive_image path: images/docs/configuration/service_account_enable_dwd.png alt: "Service Account Enable DwD" indent: 3 %}
```

Comment:
Please confirm this renders indented like this and that it's not converted to a codeblock

Reply:
Confirmed it didn't get converted to codeblock.